src/dspbase.jl

```diff
@@ -675,6 +675,25 @@ function _conv(u, v, su, sv)
     _conv!(out, u, v, su, sv, outsize)
 end
 
+function _conv_td(u::AbstractArray{<:Number, N}, v::AbstractArray{<:Number, N}) where {N}
+    output_indices = CartesianIndices(ntuple(Val(N)) do d
```

Comment:
This might be better written as a `map(axes(u), axes(v)) do ...` instead of the `ntuple`.